README.md

```diff
@@ -314,7 +314,7 @@ Run [ESLint](https://eslint.org/) to analyze the code and catch bugs:
 yarn lint
 ```
 
-Run [Prettier](https://prettier.io/) to check formatting rules:
```

Comment:
I think `Run` is fine here since people need to run Prettier to check formatting.

Reply:
Perfect

Reply:
Could you update it and put `Run` instead of `Use`?

Reply:
Updated.

Reply:
Hey! Thank you! I think that the update is not reflected here. I think you need to update the branch you used to create the PR (`patch-1`) and the changes will be reflected here automatically.